mm2src/coins/nft/nft_errors.rs

```diff
@@ -102,9 +104,9 @@ impl HttpStatusCode for GetNftInfoError {
     fn status_code(&self) -> StatusCode {
         match self {
             GetNftInfoError::InvalidRequest(_) => StatusCode::BAD_REQUEST,
-            GetNftInfoError::InvalidResponse(_) | GetNftInfoError::ParseTimestampError(_) => {
-                StatusCode::FAILED_DEPENDENCY
-            },
+            GetNftInfoError::InvalidResponse(_)
+            | GetNftInfoError::ParseTimestampError(_)
+            | GetNftInfoError::ContractTypeIsNull => StatusCode::FAILED_DEPENDENCY,
```

Comment:
Is `FAILED_DEPENDENCY` the right status code to be used if `ContractTypeIsNull`, wouldn't `NOT_FOUND` or `BAD_REQUEST` be better?

Reply:
I was choosing between `FAILED_DEPENDENCY` and `PRECONDITION_FAILED`.
bcz we expect that contract type is not null.
I dont think that `BAD_REQUEST` is right here, as we send request if NFT was found in db (one of the examples), or we itarate through txs (while we filtered null contract types previously).

Well `NOT_FOUND` may be is better, I will change it.

Reply:
Done

Reply:
`NOT_FOUND` seems to be the best choice :)